### DISCUSSION TITLE
Difference between Q. service calls

### DISCUSSION THREAD

**Discussion by shakazulu89** [serenity-is/Serenity#6748] [General]
Hey all

Can someone summarize what is the difference between Q.servicecall, Q.posttoservice, and Q.serviceRequest?
What the usage scenario for each?

And which one will trigger audit for IAuditLog rows?

thanks!

**Comment by VictorTomaili** [comment]
postToService is for html forms. 
serviceRequest is bringing the most used options as function parameters to make it easy to call serviceCall function. 
serviceCall is doing actual requests to the services. It is bringing some default values to make request to the serenity endpoints. Check the services.ts file in the source code.
